tests/functional/customer/test_alert.py

```diff
@@ -78,3 +80,33 @@ def test_can_create_a_stock_alert(self):
         alert = alerts[0]
         self.assertEqual(ProductAlert.UNCONFIRMED, alert.status)
         self.assertEqual(alert.product, product)
+
+
+class TestHurryMode(TestCase):
+
+    def setUp(self):
+        self.user = UserFactory()
+        self.product = create_product()
+
+    def test_hurry_mode_not_set_when_stock_high(self):
+        # One alert, 5 items in stock. No need to hurry.
+        stockrecord = create_stockrecord(self.product, num_in_stock=5)
+        ProductAlert.objects.create(user=self.user, product=self.product)
+
+        send_product_alerts(self.product)
+
+        self.assertEqual(1, len(mail.outbox))
+        self.assertNotIn('Beware that the amount of items in stock is limited',
+            mail.outbox[0].body)
+
+    def test_hurry_mode_set_when_stock_low(self):
+        # Two alerts, 1 item in stock. Better hurry.
```

Comment:
Both are correct English :). But I've reworded to be more straightforward. Also added tests for the case where a product has multiple stock records.